server/templates/NewChart.template.js

```diff
@@ -0,0 +1,21 @@
+import basePathConfig from '../configs/basePathConfig'
+
+const basePath = basePathConfig || ''
+
+export default () => {
+	return `
```

Comment:
can you fix the formatting of this file?

Reply:
[a826d0d](https://github.com/AMP-SCZ/dpdash/pull/78/commits/a826d0d8b81c3e20be808d5bd4546eff18343fff)